src/bootstrap.tsx

```diff
@@ -687,6 +688,10 @@ const renderApp = async () => {
       i18n.changeLanguage(defaultLanguage);
     }
 
+    const printApp = appConfig?.clientConfig?.printApp;
```

Comment:
Not entirely sure if needed, but do we need to add the new field to the [shogun-util](https://github.com/terrestris/shogun-util/blob/main/src/model/Application.ts#L53)?

Reply:
> Not entirely sure if needed, but do we need to add the new field to the [shogun-util](https://github.com/terrestris/shogun-util/blob/main/src/model/Application.ts#L53)?

The client app doesn't start without it, but maybe there is something else behind that? I am open to suggestions.